tests/test_strutil.py

```diff
@@ -23,3 +23,15 @@ def test_normal(self):
         self.assertEqual(list_or_splitstr('''a
     b        c'''), ['a', 'b', 'c'])
         self.assertEqual(list_or_splitstr('a	b	c'), ['a', 'b', 'c'])
+
+
+class test_nym(unittest.TestCase):
```

Comment:
suggestion non-blocking: Might be good to add a test that covers the '\t' case.

Reply:
Done